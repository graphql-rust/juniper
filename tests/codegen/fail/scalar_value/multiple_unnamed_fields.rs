#[derive(juniper::ScalarValue)]
enum ScalarValue {
    Variant(u32, i64),
}

fn main() {}
