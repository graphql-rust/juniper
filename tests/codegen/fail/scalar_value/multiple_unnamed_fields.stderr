error: GraphQL built-in scalars expected exactly 1 field
 --> fail/scalar_value/multiple_unnamed_fields.rs:3:12
  |
3 |     Variant(u32, i64),
  |            ^^^^^^^^^^
